scripts/react-tester.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-env node */
+
+const { execSync } = require( 'child_process' );
+const { mkdirSync, rmdirSync, copyFileSync } = require( 'fs' );
+const { resolve: resolvePath } = require( 'path' );
+const satisfiesSemver = require( 'semver/functions/satisfies' );
+const semverMinor = require('semver/functions/minor')
+
+const PACKAGE_PATH = resolvePath( __dirname, '..' );
+const TESTS_PATH = resolvePath( PACKAGE_PATH, 'react-tests' );
+
+const packageInfo = require( '../package.json' );
+const availableVersions = getVersions();
+const semverRange = getReactVersion( packageInfo );
+const versionsInRange = getVersionsInRange( semverRange, availableVersions );
+const versionsToTest = getLatestPatches( versionsInRange );
+
+try {
+	console.log( '--- Ultimate CKEditor 4 - React Integration Tester ---' );
+
+	rmdirSyncRecursive( TESTS_PATH );
+	mkdirSync( TESTS_PATH );
+
+	versionsToTest.forEach( testVersion );
+
+	rmdirSyncRecursive( TESTS_PATH );
+	console.log( 'Done without errors. Have a nice day!' );
+} catch ( error ) {
+	console.error( 'Error occured :(' );
+	console.error( error );
+	process.exit( 1 );
```

Comment:
I wonder if we could go with better error logging? For example showing which versions failed. I know it can be deducted from browsing run logs but clear information on the log end would be useful too.

Reply:
Currently the process stops if any test fails - we are not moving forward in such case, so I've just added failing version number at the end of log (below the error so it should be easily visible). Do you think we should change this workflow?

I've also limited displaying the error information while maintaining the possibility of displaying the full info using `-v` flag.